kafka_consumer/datadog_checks/kafka_consumer/client/kafka_python_client.py

```diff
@@ -29,9 +29,9 @@ def token(self):
 
 
 class KafkaPythonClient(KafkaClient):
-    def __init__(self, check, config) -> None:
+    def __init__(self, check, kafka_config) -> None:
         self.check = check
-        self.config = config
+        self.kafka_config = kafka_config
```

Comment:
I renamed this from `self.config` because `self.config` is used by the `ConfigMixin` so I wanted to avoid any possible collisions.